### ISSUE TITLE
Weird initialization using Google Collabs. 

### ISSUE THREAD

**Issue by williamsblood** [lllyasviel/Fooocus#796]
Hey I am having these errors messages when starting from collab notebook (paid version), Foocus initialize after while IDK if this behavior is intended or not it's just for your information. 

Mounted at /content/drive
Collecting pygit2==1.12.2
  Downloading pygit2-1.12.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.9/4.9 MB 48.7 MB/s eta 0:00:00
Requirement already satisfied: cffi>=1.9.1 in /usr/local/lib/python3.10/dist-packages (from pygit2==1.12.2) (1.16.0)
Requirement already satisfied: pycparser in /usr/local/lib/python3.10/dist-packages (from cffi>=1.9.1->pygit2==1.12.2) (2.21)
Installing collected packages: pygit2
Successfully installed pygit2-1.12.2
Diretório já existe
/content/drive/My Drive/Fooocus
Fast-forward merge
Update succeeded.
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
Fooocus version: 2.1.749
Installing requirements
Running on local URL:  http://127.0.0.1:7860/
Running on public URL: https://f339bbe4b3fb861970.gradio.live/

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from Terminal to deploy to Spaces (https://huggingface.co/spaces)
Total VRAM 15102 MB, total RAM 52221 MB
2023-10-26 21:36:32.316757: E tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc:9342] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2023-10-26 21:36:32.316807: E tensorflow/compiler/xla/stream_executor/cuda/cuda_fft.cc:609] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2023-10-26 21:36:32.316836: E tensorflow/compiler/xla/stream_executor/cuda/cuda_blas.cc:1518] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2023-10-26 21:36:33.321193: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Set vram state to: NORMAL_VRAM
Device: cuda:0 Tesla T4 : native
VAE dtype: torch.float32
Using pytorch cross attention
[Fooocus] Disabling smart memory
Refiner unloaded.
model_type EPS
adm 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
missi
[log.txt](https://github.com/lllyasviel/Fooocus/files/13183231/log.txt)
ng {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection'}
Base model loaded: /content/drive/My Drive/Fooocus/models/checkpoints/sd_xl_base_1.0_0.9vae.safetensors
LoRAs loaded: [('sd_xl_offset_example-lora_1.0.safetensors', 0.5), ('None', 0.5), ('None', 0.5), ('None', 0.5), ('None', 0.5)]
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.58 seconds
App started successful. Use the app with http://127.0.0.1:7860/ or 127.0.0.1:7860 or https://f339bbe4b3fb861970.gradio.live/
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 7.0
[Parameters] Seed = 1946346879500912511
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 20
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
model_type EPS
adm 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
missing {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection'}
Base model loaded: /content/drive/My Drive/Fooocus/models/checkpoints/Cópia de thinkdiffusionxl_v10.safetensors
LoRAs loaded: [('None', 0.5), ('None', 0.5), ('None', 0.5), ('None', 0.5), ('None', 0.5)]
Requested to load SDXLClipModel
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.83 seconds
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] New suffix: intricate, elegant, highly detailed, Pixiv, HDR, Stanley Artgerm Lau, WLOP, RossDraws, James Jean, Andrei Riabovitchev, Marc Simonetti, and Sakimichan, trending on
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] New suffix: extremely detailed, elegant, abstract, blue buxom, by Ilya Kuvshinov, WLOP, Ross Tran, Stanley Artgerm Lau, Ruan Jia and Fenghua Zhong. realistic
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.11 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
Preparation time: 223.09 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.02916753850877285, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.06 seconds
100% 30/30 [00:28<00:00,  1.04it/s]
Image generated with private log at: /content/drive/My Drive/Fooocus/outputs/2023-10-26/log.html
Generating and saving time: 36.03 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.02916753850877285, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.34 seconds
100% 30/30 [00:29<00:00,  1.01it/s]
Image generated with private log at: /content/drive/My Drive/Fooocus/outputs/2023-10-26/log.html
Generating and saving time: 35.24 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.66 seconds
Total time: 296.66 seconds

**Comment by williamsblood** [comment]
[log.txt](https://github.com/lllyasviel/Fooocus/files/13183236/log.txt)

**Comment by mashb1t** [comment]
Closing the issue, works with the latest version of Fooocus on Colab (paid and free).
